### PR TITLE
Enhancement: Better message from is_close

### PR DESCRIPTION
**Pull request by arounkles**
This change has `is_close` give a more informative message, including what the percentage difference was between the values.

### CODE REVIEW

File: jgt_common/assert_.py [hunk 1]
```diff
@@ -14,6 +15,16 @@ def _msg_concat(prefix, body):
     return "{} {}".format(prefix, body) if prefix else body
 
 
+def _percent_diff(a, b, precision=2):
```
Comment (rcasperson-jc) [inline jgt_common/assert_.py]: I think it'd be worth throwing this in `__init__.py` and adding a few tests for it.

Comment (arounkles) [reply jgt_common/assert_.py]: I'm cool with that. Will make that change.

Comment (rbcasperson) [reply jgt_common/assert_.py]: Pay no attention to that imposter! (Though I agree with his point)

Comment (dgou) [reply jgt_common/assert_.py]: Bump.

File: jgt_common/__init__.py [hunk 2]
```diff
@@ -1164,6 +1166,20 @@ def build_classification_rst_string(from_dict, for_module, category_name_mapping
...
+    """Get percentage difference, out to ``precision`` places."""
+    if a == b:
+        return 0
+    elif a == 0 or b == 0:
+        return nan
+    # It is debatable whether this is the correct choice for determining
+    # relative difference. min(), max(), and the mean all have a case to be made for
+    # them, both with and without abs(). See, e.g.:
+    # https://en.wikipedia.org/wiki/Relative_change_and_difference#Definitions
+    return round(abs((a - b) / max(abs(a), abs(b))) * 100, precision)
```
Comment (arounkles) [inline jgt_common/__init__.py]: To add to this comment, I feel like the `precision` parameter might not really be all that useful. Perhaps adding a choice of scale functions, similar to [this npm package](https://www.npmjs.com/package/math-relative-difference) would be a better choice. Also like that package, perhaps we should return the relative difference directly, and let the user multiply by 100 to get a percentage value.

Comment (dgou) [reply jgt_common/__init__.py]: I kinda like that idea, except I don't like the string or function aspect of it.
Probably I would favor just always having a function, provide a bunch of nicely named functions to choose from (vs. having them encoded as strings) with a default to match the code you already have?

Comment (arounkles) [reply jgt_common/__init__.py]: So, have, say, an enum of a bunch of scale functions and then to call it with one of the others, something like:

```
jgt_common.percent_diff(3, 2, scale_func=mean_abs)
```

This lets users pass in their own oddball scale func if they want.

**Comment by arounkles** [comment]
@dgou I had thought about it, but never got any feedback beyond yours, saying that maybe do it if there's any interest. So far I haven't seen any interest in the PR at all since then. :expressionless:

**Pull request merged**